lib/math.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var R = require('ramda');
+
+var operators = {
+  '+': R.add,
+  '-': R.subtract,
+  '*': R.multiply,
+  '/': R.divide,
+  '%': R.modulo,
+  '++': R.inc,
+  '--': R.dec
+};
```

Comment:
That page says it's an experimental technology, and Ramda lacks a helper for it (unless I missed it). But I'd welcome a PR adding the functionality. 😁